spec/support/article_helper.rb

```diff
@@ -1,34 +1,34 @@
 module ArticleHelper
-	def add_new_article(article)
-		visit new_article_path
+  def add_new_article(article)
+    visit new_article_path
 
     fill_in 'Name', with: article.name
     fill_in 'Url', with: article.url
     fill_in 'Description', with: article.description
     click_button 'Submit'
-	end  
+  end
+
+  def edit_article(articleid)
+    visit article_path(articleid)
+    click_button 'Edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.